doc/classes/Quat.xml

```diff
@@ -132,26 +132,6 @@
 				Returns a copy of the quaternion, normalized to unit length.
 			</description>
 		</method>
-		<method name="set_axis_angle">
-			<return type="void">
-			</return>
-			<argument index="0" name="axis" type="Vector3">
-			</argument>
-			<argument index="1" name="angle" type="float">
-			</argument>
-			<description>
-				Sets the quaternion to a rotation which rotates around axis by the specified angle, in radians. The axis must be a normalized vector.
-			</description>
-		</method>
-		<method name="set_euler">
-			<return type="void">
-			</return>
-			<argument index="0" name="euler" type="Vector3">
-			</argument>
-			<description>
-				Sets the quaternion to a rotation specified by Euler angles (in the YXZ convention: when decomposing, first Z, then X, and Y last), given in the vector format as (X angle, Y angle, Z angle).
-			</description>
-		</method>
```

Comment:
Missing bindings, or removed on purpose?

Reply:
Commented out in `variant_call.cpp`:
```
// FIXME: Quat is atomic, this should be done via construcror
//ADDFUNC1(QUAT, NIL, Quat, set_euler, VECTOR3, "euler", varray());
//ADDFUNC2(QUAT, NIL, Quat, set_axis_angle, VECTOR3, "axis", FLOAT, "angle", varray());
```

Reply:
It seems that #30744 finally got resolved in core :)